src/vs/gitpod/node/gitpodInsightsAppender.ts

```diff
@@ -214,6 +215,31 @@ function mapTelemetryData(eventName: string, data: any): RemoteTrackMessage | un
 					timestamp: data.timestamp
 				},
 			};
+		case 'gettingStarted.ActionExecuted':
```

Comment:
It would be nice if there is one event/table to see all activities around getting started, i.e. `vscode_getting_started`.